recipes/archive/justfile

```diff
@@ -11,7 +11,10 @@ tree     := join('archive', 'pc', 'mod')
 
 @pack NAME:
     {{wk_cli}} pack '{{ join(source, NAME, "source", "archive") }}' -o '{{ join(source, NAME) }}'; \
-    just rename '{{ join(source, NAME, "archive.archive") }}' '{{ join(source, NAME + ".archive") }}';
+    just rename '{{ join(source, NAME, "archive.archive") }}' '{{ join(source, NAME + ".archive") }}'; \
+    just xl '{{NAME}}';
+
+@xl NAME:

```

Comment:
since you follow the same naming convention, you don't need this anymore,
please revert it .